### DISCUSSION TITLE
Regarding the vulnerabilities display under NodeJS.

### DISCUSSION THREAD

**Discussion by mightysai1997** [eslint/eslint#18522] [Q&A]
Hi,

We can showcase both JavaScript and Node.js vulnerabilities within Node.js ?

Please let me know if there's any reason why we can't display them under Node.js.

**Comment by Rec0iL99** [comment]
Hi @mightysai1997, is this something related to ESLint?

**Comment by mightysai1997** [comment]
Hi @Rec0iL99 , Have a good day.

Yes, it is related to ESLint.

**Reply by Rec0iL99** [reply]
Could you please elaborate more on what you are looking for so we can help you here?

**Comment by mightysai1997** [comment]
Yes, I mean what the result I got after the scan, Can I store it under
Node. JS file or else I need to store under Javascript file?

**Reply by Rec0iL99** [reply]
If you are looking to store the linting results from ESLint in another file then you can do this:

```bash
> npx eslint index.js > results
```
